python/tvm/relax/training/optimizer.py

```diff
@@ -0,0 +1,571 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+"""Provide abstraction for defining optimizers and a set of common optimizers."""
+
+from typing import List, Union
+
+import numpy as np
```

Comment:
Now mypy reports error on this line:
```
error: Cannot find implementation or library stub for module named "numpy"
```
But our optimizer really needs numpy to create the `tvm.ndarray` that is used in the optimization process. How could we handle this?